### PR TITLE
Pub-949 outbound api

### PR DESCRIPTION
**Pull request by kurt-uk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-949

### Change description ###

added the calls to channel management and publication services for api third parties

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/ChannelManagementService.java [hunk 1]
```diff
@@ -19,6 +20,7 @@
 public class ChannelManagementService {
 
     private static final String EMAIL_PATH = "/channel/emails";
+    private static final String API_PATH = "/channel/emails";
```
Comment (joshblackmoor) [inline src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/ChannelManagementService.java:23]: Should this not be /channel/api ?

File: src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java [hunk 2]
```diff
@@ -192,14 +191,18 @@ private void duplicateSubscriptionHandler(Subscription subscription) {
...
-        channelManagementService.getMappedEmails(subscriptionsList).forEach((email, listOfSubscriptions) -> {
-            SubscriptionsSummary summaryToSend =
-                formatSubscriptionsSummary(artefactId, email, listOfSubscriptions);
-
-            log.info("Summary being sent to publication services: " + summaryToSend);
-
-            publicationServicesService.postSubscriptionSummaries(summaryToSend);
-        });
+        List<Subscription> emailList = sortSubscriptionByChannel(subscriptionsList, Channel.EMAIL.notificationRoute);
+        List<Subscription> apiList = sortSubscriptionByChannel(subscriptionsList, "API");
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java:195]: I think this should be Channel.API.notificationRoute

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/ChannelManagementService.java (hunk 1 above)
Comment (kurt-uk) [reply src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/ChannelManagementService.java:23]: 👍🏼

File: src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java (hunk 2 above)
Comment (kurt-uk) [reply src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java:195]: updated
was going to do this however its not .API.notificationRoute, its API_COURTEL.notification route and didnt wanna confuse devs thinking they had to add API_OTHER.notificaiton route, i could just see duplicated code happening 😂 updated anyway and hoping for the best ;)

**Review by joshblackmoor — APPROVED** [review]
LGTM

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**